examples/with-supabase/.gitignore

```diff
@@ -28,6 +28,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.pnpm-debug.log*
```

Comment:
(maybe add a note in the PR to indicate why (auth-pages) was removed/renamed?)

Reply:
Fixed.